src/view/cards/Certification/__tests__/Certification.spec.tsx

```diff
@@ -0,0 +1,36 @@
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+import { render, screen } from 'utils/testing';
+import { mockCerts } from 'services/api/__mocks__';
+import { CertificationCard } from '../Certification';
+
+describe('<Certificates />', () => {
+  const { data: CertificateData } = mockCerts;
+  it('renders', () => {
+    const { container } = render(
+      <CertificationCard certification={CertificateData[0]} />,
+    );
+    expect(container).toMatchSnapshot();
+  });
+  it('Checks that the more info button works correctly', () => {
+    render(<CertificationCard certification={CertificateData[0]} />);
+    userEvent.click(screen.getByText('More Info'));
+    expect(screen.getByRole('button', { name: 'More Info' })).toHaveAttribute(
+      'href',
+      `/certifications/${CertificateData[0].id}`,
+    );
+  });
+  it('Checks for expired date and checks the claimed logic', () => {
+    render(<CertificationCard certification={CertificateData[0]} />);
+    expect(screen.getByText('Expired on 8/18/2020'));
+    expect(screen.getByText('Claimed'));
+  });
+  it('Checks for valid expiration date and checks the unclaimed logic', () => {
+    CertificateData[0].valid_to = 2217110400;
```

Comment:
So this line is currently modifying the `CertificateData` object that is shared by all of the other tests. If we were to write another test after this one that relied on the `valid_to` property, it would be dependent upon this test modifying that prop, which is not ideal. 

Something like this will allow us to make a copy of the `CertificateData` that we can modify however we want without affecting other tests.
```suggestion
    const expiredCert = { ...CertificateData, valid_to: 2217110400, assertion_id: 'test' }
```

Reply:
However, we only want to test one change at a time, so in this case it should probably just be 

```ts
    const expiredCert = { ...CertificateData, valid_to: 2217110400 }
```